docs/rfcs/001-updateable-bundled-packages.md

```diff
@@ -0,0 +1,100 @@
+# Updateable Bundled Packages
+
+## Status
+
+Proposed
+
+## Summary
+
+> One paragraph explanation of the feature.
+
+This feature will enable an opt-in subset of bundled Atom packages to be updated with `apm` outside of the Atom release cycle.  This will enable users to receive new functionality and bug fixes for some bundled packages as regularly as needed without waiting for them to be included in a new Atom release.  This is especially important for packages like [GitHub](https://github.com/atom/github/) and  [Teletype](https://github.com/atom/teletype/) which provide essential Atom functionality and could be improved independently of Atom.
+
+## Motivation
+
+> Why are we doing this? What use cases does it support? What is the expected outcome?
+
+Atom currently uses a monthly release cycle with staged Stable and Beta releases so that major issues get caught early in Beta before reaching the Stable release.  Because Atom releases updates monthly, this means that a new feature merged into `master` right after a new Atom release could take one month to reach the next Beta and then another month to reach Stable.
```

Comment:
> Because Atom releases updates monthly, this means that a new feature merged into `master` right after a new Atom release could take one month to reach the next Beta and then another month to reach Stable.

This is the primary reason that Teletype is not bundled with Atom. Since its initial launch, we've shipped a Teletype release almost every week, delivering new features and bug fixes as soon as they're ready, as opposed to waiting up to 2 months to get each release into users' hands.

Reply:
:+1: